src/components/Checkout/Checkout.js

```diff
@@ -8,10 +21,188 @@ const stripePromise = loadStripe(
 		''
 );
 
-export default function Checkout() {
+const Checkout = () => {
+	const [clientSecret, setClientSecret] = useState('');
+
+	useEffect(() => {
+		// Create PaymentIntent as soon as the page loads
+		axios
+			.post('http://localhost:4242/api/checkout/create-payment-intent', {
```

Comment:
It does matter because this line of code would work only for you and not others who will be using the default 5000 port. Now would be a good time to merge main into your branch and make sure you're using the `axiosInstance` that is currently in the main branch.